src/table/utils/styling-utils.js

```diff
@@ -122,19 +149,31 @@ export function getColumnStyle(styling, qAttrExps, stylingInfo) {
   return {
     ...styling,
     color: columnColors.cellForegroundColor || styling.color,
-    background: columnColors.cellBackgroundColor,
+    backgroundColor: columnColors.cellBackgroundColor,
   };
 }
 
-export function getSelectionColors(cell, selectionState, background, tableBackgroundColor) {
+/**
+ * Get different styles for cells based on wether they are
+ * selected or possible to be selected or not possible to be selected
+ * @param {Object} cell - each cell in the table body
+ * @param {Object} selectionState - the selected cell
```

Comment:
these descriptions are off. `cell` isn't each cell, it is one cell, since we call this fore every cell. `selectionState` contains info about what cells are selected

Reply:
Good. Updated